bamboo/Dockerfile

```diff
@@ -84,5 +85,9 @@ RUN update-alternatives --set java /usr/lib/jvm/java-8-openjdk-arm64/bin/java
 # Add file in /artemis/bin/mvn that uses the correct java version and passes all arguments to mvn
 RUN mkdir /artemis && mkdir /artemis/bin && printf '#!/bin/bash\nJAVA_HOME=/usr/lib/jvm/java-17-oracle /usr/bin/mvn "$@"\n' > /artemis/bin/mvn && chmod 777 /artemis/bin/mvn
 
+# Add the newest gradle version
```

Comment:
I think we should lock the Gradle version here.
The versions for some of the other dependencies are also locked and Gradle is known to have some minor incompatibilities between the versions, which I would try to avoid.
I would at least fix the major version (maybe also the minor, as we did it for the other dependencies)

Reply:
Changed in 003d7e9, version is freezed to 7.3.